fastdl.py

```diff
@@ -98,13 +113,15 @@ def main():
 						if not os.path.exists(fulldir):
 							print("Directory {} wasn't found on fastdl path, creating...".format(fulldir))
 							os.makedirs(fulldir)
-
 						rootfile = os.path.join(dirpath, file)
-						#special case for files bigger then 150mb
+						if BlackListedFiles != [] and str(file) in BlackListedFiles:
+							print("Found {} which is blacklisted, ignoring...".format(str(file)))
+							continue
+						#special case for files bigger than 150MB
 						if os.path.getsize(rootfile) < (150 * 1024 * 1024):
-							addToFastdl(rootfile, os.path.join(fulldir, "{}.bz2".format(file)))
+							addToFastdl(rootfile, os.path.join(fulldir, "{}.bz2".format(file)), BlackListedFiles)

```

Comment:
Why passing BlackListedFiles as an argument here if you aren't using it in the function itself?